### ISSUE TITLE
Missing dependency

### ISSUE THREAD

**Issue by selpoG** [kenkangxgwe/lsp-wl#22]
I got following error:
```
Get::noopen: Cannot open GitLink`.

Needs::nocont: Context GitLink` was not created when Needs was evaluated.
```
This error has resolved by installing [GitLink](https://github.com/WolframResearch/GitLink).
This dependency should be written in `README.md`, I think.

**Comment by kenkangxgwe** [comment]
GitLink comes w/ MMA after a certain version, so I didn't check it for dependency. Since it is not a requirement, I will mark it as an optional dependent which will not give any warning and skip the update checking.

**Comment by kenkangxgwe** [comment]
Fixed in 133ffd523cde000877e8c08faa5d2ea7d4ac55fa.
